docs/api/users/create.md

```diff
@@ -1,7 +1,187 @@
+      shasum.update(password)
+      shasum.update(passwordSalt.toString('hex'))
+      const passwordHash = shasum.digest('hex')
+
+      resolve({
+        "passwordSalt": passwordSalt.toString('hex'),
+        "passwordHash": passwordHash
+      })
+    })
+  }
+
+  (async () => {
+    const openhimOptions = {
+      apiURL: 'https://localhost:8080',
+      apiEndpoint: '/users',
+      username: 'root@openhim.org',
+      password: 'openhim-password',
+      rejectUnauthorized: false
+    }
+
+    const userPassword = "user-password"
+    const userPasswordDetails = await genUserPassword(userPassword)
+
+    const SampleData = 'SampleData'
+
+    const headers = await genAuthHeaders(openhimOptions)
+
+    const options = { 
+      method: 'POST',
+      url: `${openhimOptions.apiURL}${openhimOptions.apiEndpoint}`,
+      rejectUnauthorized: openhimOptions.rejectUnauthorized,
+      headers: headers,
+      body: SampleData,
+      json: true
+    }
+
+    request(options, (error, response, body) => {
+      if (error) throw new Error(error)
+
+      console.log({
+        statusCode: response.statusCode,
+        body
+      })
+    })
+  })()
+  ```
+
+  Execute the below command in your terminal to run the nodejs script
+
+  ```bash
+  node openhim-api.js
+  ```
+
+  </TabItem>
+  <TabItem value="bash">
+
+  Ensure that you have created your bash script to construct the HTTP authentication headers and send the request to the OpenHIM API as described in the [authentication section](../introduction/authentication.md).
+
+  To create a user, you first need to generate a password hash and salt. The script below can be used for this
+
+  ```curl
+  # gen-user-password.sh
+
+  #!/bin/bash
+  if (( $# < 1)); then
+      echo "OpenHIM User Password: Curl wrapper that generates the user Password details";
+      echo "Usage: $0 PASSWORD";
+      exit 0;
+  fi
+  pass=$1;
+  shift;
+  shift;
+
+  salt=`cat /dev/urandom | tr -dc 'abcdef0-9' | fold -w 32 | head -n 1`;
+  passhash=`echo -n "$pass$salt" | shasum -a 512 | awk '{print $1}'`;
+
+  echo Password Salt: $salt
+  echo Password Hash: $passhash
+
+  echo "";
+  ```
+
+  Copy and execute the bash script above with the below command. Substitute the <USER_PASSWORD> with your desired password before executing.
+
+  ```curl
+  ./gen-user-password.sh <USER_PASSWORD>
+  ```
+
+  Execute the below command in your terminal where the file is located with the required arguments to create the user. Replace the placeholder arguments with the correct implementation details and ensure your `SampleData.json` file exists with the user object. Add the generated passwordHash and passwordSalt to the `SampleData.json` before executing the command.
+
+  ```curl
+  ./openhim-api.sh root@openhim.org openhim-password -v https://localhost:8080/users -d @SampleData.json -H "Content-Type:application/json"
+  ```
+
+  </TabItem>
+  </Tabs>
+
+  The response status will be 201 if successful.
```

Comment:
the status values needs to be within markdown quotes so its gets highlighted a bit 
`201`